### ISSUE TITLE
Exquilla makes Thunderbird extremely slow

### ISSUE THREAD

**Issue by Rik44444** [rkent/exquilla#21]
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:61.0) Gecko/20100101 Firefox/61.0
Build ID: 20180807170231

Steps to reproduce:

i downloaded Thunderbird, added some normal accounts, then discovered i could not do that for Exchange, then downloaded Exquilla addon, and added an Exchange account. I'm also using TBsync. 

Actual results:

using Exquilla greatly slowed Thunderbird, to a degree that after typing, it would take several seconds for words to appear, and there being a delay of seconds after clicking on a button. after i disabled Exquilla, everything went back to normal

Expected results:

i would like to be able to type and click buttons without a delay of seconds :) 
ps why is something like Exquilla not built into THunderbird? Would make sense to include a possibility for exchange accounts!

--

in conclusion: developers thanks for producing this great addon, so valuable!! except now i deleted it again, because THunderbird becomes so slow it becomes unworkable. once this is corrected, ill happily use it - it should be standard in Thunderbird.

**Comment by rkent** [comment]
At this point in time, no further releases of ExQuilla are planned. I have largely retired from this sort
of software development due to a variety of issues, including the difficulty of adopting ExQuilla to future
releases of Thunderbird, as well as some personal medical issues.

Recording of issues here may be useful to someone in the future who tries to take over ExQuilla or
develop a substitute of some sort, but there are no current plans to followup with issues here to fix bugs
or incorporate suggested changes.

**Comment by benbucksch** [comment]
Compare
https://bugzilla.mozilla.org/show_bug.cgi?id=441710
https://bugzilla.mozilla.org/show_bug.cgi?id=733039

**Comment by bon-ami** [comment]
to repair folders sound nice
